asm/macros/event.inc

```diff
@@ -315,7 +317,8 @@
 	.byte 0x2d
 	.endm
 
-	@ Sets the values of variables 0x8000, 0x8001, and 0x8002 to the current hour, minute, and second. In FRLG, this command sets those variables to zero.
+	@ Sets the values of variables 0x8000, 0x8001, and 0x8002 to the current hour, minute, and second. In FRLG, 
+        @ this command sets those variables to zero.
```

Comment:
Indent `.s`, `.inc` files with tab character, not spaces.

Reply:
Updated with fix, I used the github editor, which doesn't automatically fix mistakes like vscode does. My bad.